blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,393 @@
+.v2-footer-wrapper {
+    background: var(--c-primary-black);
+  }
+  
+  .v2-footer {
+    color: var(--c-secondary-silver);
+    padding: 60px 32px 10px;
+    font-size: 13px;
+    margin: auto;
+    max-width: 1376px;
+  }
+  
+  .v2-footer ul {
+    list-style: none;
+    padding: 0;
+  }
+  
+  .v2-footer .v2-footer-social-media-section {
+    margin-top: 21px;
+    display: flex;
+    gap: 12px;
+    flex-wrap: wrap;
+  }
+  
+  .v2-footer .v2-footer-social-media {
+    border-radius: 17.5px;
+    color: var(--c-tertiary-cool-gray);
+    text-decoration: none;
+    display: flex;
+    width: 35px;
+    height: 35px;
+    align-items: center;
+    justify-content: center;
+    font-size: 20px;
+  }
+  
+  .v2-footer .v2-footer-social-media:hover {
+    color: var(--c-tertiary-cool-white);
+  }
+  
+  .v2-footer .v2-footer-social-media::before {
+    width: 17px;
+  }
+  
+  .v2-footer h1,
+  .v2-footer h2,
+  .v2footer h3 {
+    font-size: 14px;
+    font-weight: 400;
+    margin-bottom: 5px;
+    border-bottom: 2px solid;
+    line-height: 22px;
+    margin-top: 0;
+  }
+  
+  .v2-footer ul li a {
+    color: var(--c-secondary-silver);
+  }
+  
+  .v2-footer .v2-footer-bottom-section li a {
```

Comment:
Combine this selector with the one above it

Reply:
Updated